.github/ISSUE_TEMPLATE/release_new_version.md

```diff
@@ -0,0 +1,70 @@
+---
+name: 'For maintainers: Release a new version'
+about: This is the Developer-only template which we use for tracking the process of future releases.
+title: 'Prepare Release 3.y.z'
+labels: release
+assignees: ''
+---
+
+**Target timeline**
+Schuled feature freeze / Start of translation process: <!-- Add a date, usually about 6-8 weeks from the last release, e.g. 2021-04-20 -->
+Approximate release date: <!-- Usually 2-3 weeks after starting translations, e.g. 2021-05-04 -->
+
+**Checklist**
+- [ ] Assign this issue to the release shepherd who is in charge of managing this checklist
+- [ ] Create a milestone and add all relevant issues
+- [ ] Ensure that all issues/PR targeted for this release are done by checking the milestone
+- [ ] Declare a code freeze. PRs can still be worked on and may get reviewed, but must not be merged unless agreed explicitly.
+- [ ] Write the Changelog
+- [ ] [Tag a beta release](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#steps-for-a-specific-release)
+- [ ] Announce the beta release on Github Discussions
+- [ ] Get feedback on the stability of the beta release
+- [ ] Website translations
+  - [ ] Create new branch `translate3_y_z` based on branch `release`
+  - [ ] Squash-merge `changes` into `translate3_y_z`
+  - [ ] Ping translators. Translators should open PRs with their changes against the `translate3_y_z` branch (?) and reference this issue ("Add German translations for 3.y.z")
+  - [ ] Wait for all PRs to be merged
+  - [ ] Merge `translate3_y_z` into release
+- [ ] App installer translations
+  - [ ] Ping translators
```

Comment:
Would this have been done as part of the previous "ping" (ie setting up the Issue for them to link their PRs to)?

Reply:
Yes, I think App translations and App installer translations should now be handled with a single Issue per language. Installer translations will also be updated less often, I think. I have combined that now.